ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/LegalRequirements/LegalModelBase.cs

```diff
@@ -10,27 +9,24 @@ namespace ApplyToBecomeInternal.Pages.TaskList.LegalRequirements
 {
 	public class LegalModelBase : PageModel
 	{
-		private readonly IAcademyConversionProjectRepository _academyConversionProjectRepository;
-		protected readonly ILegalRequirementsRepository LegalRequirementsRepository;
+		protected readonly IAcademyConversionProjectRepository AcademyConversionProjectRepository;
```

Comment:
The naming of this field doesn't follow the convention we've been using.
Should be - _academyConversionProjectRepository